share/zsh/site-functions/_brew

```diff
@@ -145,4 +150,6 @@ case "$words[1]" in
       _brew_outdated_formulae
       _wanted outdated_formulae expl 'outdated formulae' compadd -a outdated_formulae
     fi ;;
+  *)
+    __brew_command "$command"
```

Comment:
Another minor thing for consistency: Could you please add a trailing `;;` to terminate this case like all the others? (I know it's not strictly necessary.)